docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -51,6 +51,25 @@ Note that these values are ONLY used to launch instances. After launch, the
 manager will query the AWS API to find the instances of each type that have the
 ``runfarmtag`` set above assigned to them.
 
+``launch_instances_timeout_minutes``
+""""""""""""""""""""""""""""""""""""
+
+Integer number of minutes that the ``launchrunfarm`` command will attempt to
+request new instances before giving up.  This limit is used for each of the types
+of instances being requested.  For example, if you set to 60,
+and you are requesting all four types of instances, ``launchrunfarm`` will try
+to launch each instance type for 60 minutes, possibly trying up to a total of
+four hours.
+
+This limit starts to be applied from the first time ``launchrunfarm`` receives a
+``ClientError`` response in all AvailabilityZones for your region.  In other words,
```

Comment:
Nit: Can you say AvailabilityZones (AZs)

Reply:
Sure.  
```suggestion
``ClientError`` response in all AvailabilityZones (AZs) for your region.  In other words,
```